core/src/main/java/com/linecorp/armeria/common/MediaTypeNames.java

```diff
@@ -471,11 +471,14 @@
      * {@value #ZIP}.
      */
     public static final String ZIP = "application/zip";
-
     /**
      * {@value MULTIPART_FORM_DATA}.
      */
     public static final String MULTIPART_FORM_DATA = "multipart/form-data";
+    /**
+     * {@value #GRAPHQL}.
+     */
+    public static final String GRAPHQL = "application/graphql";
```

Comment:
I should've added `MULTIPART_FORM_DATA` in alphabetical order. 
Could you align `MULTIPART_FORM_DATA` and `GRAPHQL` in alphabetical order?